framework/src/postprocessors/SteadyStateRelativeChangeNorm.C

```diff
@@ -0,0 +1,36 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "SteadyStateRelativeChangeNorm.h"
```

Comment:
the name is good in your context, but not in the general context where users might not be seeking steady state

How about VariableChangeOverTimePostprocessor ? Re-uses the same pattern as other PPs

Reply:
How about `SolutionChangeOverTimePostprocessor`? This postprocessor is looking at the entire nonlinear/aux system, not just a single variable (which I think might be implied by your suggestion).

Reply:
so this defaults to the system of interest? eg the same being checked for SS convergence?

Reply:
yeah, it just fetches `_solution_change_norm`, so it is consistent with the Executioner settings

Reply:
that does make it really consistent with this "SteadyState" search. 

SolutionChangeOverTimePostprocessor works for me, and the doco can let the user know about that. 
Maybe one day people will want to select which system and they ll just need to add an enum here